CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Dev Edmonton Society is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, religion (or lack thereof) and any personal characteristics protected under [the Alberta Human Rights Act](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=4&ved=2ahUKEwjcx9-av8LdAhWnxVQKHcBsAOQQFjADegQICxAC&url=http%3A%2F%2Fwww.qp.alberta.ca%2Fdocuments%2FActs%2FA25P5.pdf&usg=AOvVaw2SRTsLqFGPFeD2iQ9MC9AJ).
```

Comment:
Is contributors the right noun for us?

Reply:
Maybe members or community members?

Reply:
@benzittlau I'm not 100% sold on it, but wasn't really sure what else to use. Members isn't right as not all people engaging with us are members. Examples of people this would apply to are:

* meetup and event attendees
* Slack users
* meetup organizers
* board members

`People` might work? As in, `A primary goal of Dev Edmonton Society is to be inclusive to the largest number of people`. It's also in our mission which is a nice symmetry.

Reply:
I noticed "community members" is used elsewhere in the document, any concerns with that?

Reply:
> I noticed "community members" is used elsewhere in the document, any concerns with that?

Can you tell that I copied this extensively from the PythonConf CoC? I'll update those too.